config/environments/production.rb

```diff
@@ -37,7 +37,7 @@
   # config.active_storage.service = :local
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
Might want an if statement around this depending on if TLS is being used or not

Reply:
This is a bad idea, because without an actual certificate (which you don't set up) this will fail. Also consider the situation when in non-fedramp environment the certificate is not available at all...

Reply:
For the certificate, I was originally setting that up in `0_clowder-config.rb` by assigning `SSL_CERT_FILE` to the cert file but apparently this breaks clowdwatch logging when tested in fedramp so I'm trying to figure out how to verify the certificate during runtime

I'll also find a different way to make sure rails uses HTTPS(when it needs to)

Reply:
There are 2 certificates, one is the CA for verifying other connections towards other services (rbac, ssg, etc) and one for being able to listen in an encrypted fashion in rails. Please make sure that you understand the difference between those two and know which is which.  If this is not clear in the clowder documentation, please reach out to Gábor or Pete.